notebooks/official/pipelines/automl_tabular_classification_beans.ipynb

```diff
@@ -2,13 +2,13 @@
   "cells": [
```

Comment:
<p>explain the parameters</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/787/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>